File: Iniciante/URI 1009.hs
import Text.Printf

main :: IO ()
main = do
    nome' <- getLine
    salarioFixo' <- getLine
    vendas' <- getLine

    let nome = read nome' :: String
    let salarioFixo = read salarioFixo' :: Double
    let vendas = read vendas' :: Double
    let total = ((vendas * 15) / 100) + salarioFixo

    putStrLn $ printf "TOTAL = R$ %.2f" total

File: Iniciante/URI 1009.php
<?php
    $nome = trim(fgets(STDIN));
    $salarioFixo = trim(fgets(STDIN));
    $vendas = trim(fgets(STDIN));
    $total = (($vendas * 15) / 100) + $salarioFixo;
    $total = number_format($total, 2, ".", "");
    echo "TOTAL = R$ $total\n";
?>